specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-11-01/costmanagement.json

```diff
@@ -1098,6 +1098,232 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.CostManagement/getReservationDetailsReport": {
+      "post": {
+        "tags": [
+          "ReservedInstances"
+        ],
+        "operationId": "GetReservationDetailsReport_ByBillingAccountId",
+        "description": "Lists the reservations details for provided date range asynchronously based on enrollment id.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/costmanagement/"
+        },
+        "x-ms-examples": {
+          "ReservationDetails": {
+            "$ref": "./examples/GetReservationDetailsReportByBillingAccount.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/enrollmentIdParameter"
+          },
+          {
+            "$ref": "#/parameters/startDateParameter"
+          },
+          {
+            "$ref": "#/parameters/endDateParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. Request will be processed. Use the Operation-Location header to check the status.",
+            "headers": {
+              "Operation-Location": {
+                "description": "The URL to check the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                "type": "integer"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.CostManagement/getReservationDetailsReport/status/{statusId}": {
```

Comment:
You typically do not need to document the operation status APIs in swagger as the original operation is marked async and should be following the standard async pattern linked earlier.  Is there a special requirement here?

Reply:
If you decide you need this in your SDKs the URI should follow the normal operation status conventions described here:  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#202-accepted-and-location-headers

Reply:
I will check with PM if we need to document the operation status API and also make the necessary changes to follow the Operation status conventions. Thanks

Reply:
updated